restricted_site_access.php

```diff
@@ -102,7 +102,7 @@ public static function add_actions() {
 	public static function ajax_notice_dismiss() {
 
 		// @codeCoverageIgnoreStart
-		if ( ! defined( 'WP_TESTS_DOMAIN' ) ) {
+		if ( ! defined( 'WP_PHP_TESTS_DOMAIN' ) ) {
```

Comment:
Not a blocker, but I feel this could be named in a more semantic way as PHP Unit won't be running on a domain. Something like `PHP_UNIT_TESTS_ENV` or `PHP_UNIT_ENV`?